tests/db/model/aspect/update.js

```diff
@@ -92,155 +88,136 @@ describe('db: aspect: update: ', () => {
         done(new Error('expecting it to be updated'));
       }
     })
-    .catch((err) => done(err));
+    .catch(done);
   });
 
   it('update name of an aspect', (done) => {
     Aspect.findOne({ where: { name: u.name } })
     .then((o) => {
-      // console.log(o.dataValues);
       expect(o.dataValues).to.have.property('name').to.equal(u.name);
       return o.update({ name: 'newName' });
     })
     .then((o) => {
       expect(o.dataValues).to.have.property('name').to.equal('newName');
-      // console.log(o.dataValues);
       done();
     })
-    .catch((err) => done(err));
+    .catch(done);
   });
 
   it('update description of an aspect', (done) => {
     Aspect.findOne({ where: { name: u.name } })
     .then((o) => {
       const descriptionDefault = o.dataValues.description;
-      // console.log(o.dataValues.description);
       expect(o.dataValues).to.have.property('description')
       .to.equal(descriptionDefault);
       return o.update({ description: 'newDescription' });
     })
     .then((o) => {
       const newDefault = o.dataValues.description;
-      expect(o.dataValues).to.have.property('description').to.equal(newDefault);
-      // console.log(o.dataValues.description);
+      expect(o.dataValues).to.have.property('description', newDefault);
```

Comment:
Since newDefault === o.dataValues.description,
we can check for description only
expect(o.dataValues).to.have.property('description'),